BeardedManStudios/Source/Forge/Networking/CommonServer.cs

```diff
@@ -27,11 +27,11 @@ public bool PlayerIsReceiver(NetworkingPlayer player, FrameStream frame, float p
 					return false;
 			}
 
-            // check if sender is null as it doesn't get sent in certain cases
-            if (frame.Sender != null)
-            {
-                PlayerIsDistanceReceiver(frame.Sender, player, frame, proximityDistance, proximityModeUpdateFrequency);
-            }
+			// check if sender is null as it doesn't get sent in certain cases
```

Comment:
Another tiny nitpick, the rest of the file uses spaces, though the rest of the classes in the Networking folder seem to be inconsistent when it comes to spaces vs tabs - maybe convert this entire file to tabs and fix indentation?

Reply:
yup sounds good. that will get it inline with the contrinbution guide too.